include/deal.II/fe/mapping_fe.h

```diff
@@ -604,6 +607,18 @@ MappingFE<dim, spacedim>::preserves_vertex_locations() const
   return true;
 }
 
+
+template <int dim, int spacedim>
+bool
+MappingFE<dim, spacedim>::is_compatible_with(
+  const ReferenceCell::Type &cell_type) const
+{
+  if (cell_type.get_dimension() != dim)
+    return false; // TODO: or is this an error?
+
+  return true;
```

Comment:
Isn't it only `true` if the finite element used by this `MappingFE` builds on the given reference cell?